### ISSUE TITLE
Manually split training and test frames

### ISSUE THREAD

**Issue by Skydddoogg** [DeepLabCut/DeepLabCut#892]
Usually, after getting extracted frames from videos, the collection of video frames will be automatically split into training and test set via create_training_dataset function. However, I would like to select which frames will be in the training set and test set by myself. Are there solutions to do so without modifying the create_training_dataset function and creating a new one?

**Comment by AlexEMG** [comment]
Yes, you can pass the train/test index yourself. Check out the docstrings of create_training_dataset please, e.g. https://github.com/DeepLabCut/DeepLabCut/blob/2d813d96277b84dae4256b1465e76e79fd9b0bb3/deeplabcut/generate_training_dataset/trainingsetmanipulation.py#L722
